app.js

```diff
@@ -1,75 +1,91 @@
 
 //Greeting
 var userName = prompt ('What\'s your name?');
-if(userName == null || userName === ""){
-  userName = "guest";
-}
+// if(userName == null || userName === ''){
```

Comment:
Good idea to get rid of commented out code like this before submission (unless it's commented code you want to re-use later or that helps remind you something).